AWS/SecretsManager/SecretsRepositoryFactory.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace ToucanTech\AWS\SecretsManager;
+
+use Aws\SecretsManager\SecretsManagerClient;
+
+class SecretsRepositoryFactory
+{
+
+    const DEFAULT_AWS_REGION = 'eu-west-1';
+
+    public static function getSecretsManagerRepository($region = self::DEFAULT_AWS_REGION)

```

Comment:
:bookmark: keep type hinting consistency when using across other files.

Reply:
:bookmark: Note more than anything as this might be down to style or practice or patterns that are followed. The way we are using this factory doesn't necessarily need it to be static and could be a more conventional class with using Construct.